### ISSUE TITLE
Non-critical error after sync

### ISSUE THREAD

**Issue by frafra** [Adambean/gitlab-ce-ldap-sync#10]
```
[error] Gitlab failure: "namespace.route" can't be blank, "notification_email" can't be blank, "notification_email" is invalid
```

**Comment by Adambean** [comment]
I guess this is for new users being created?
Which version of Gitlab are you running? (I've not tried this tool on 12 yet.)

**Comment by frafra** [comment]
Yes, it happens after users creation. I am testing it with GitLab 12.

**Comment by Adambean** [comment]
I just ran this against our updated Gitlab 12 instances today. A single new user needed to be created, and did so without any problems.

Are you certain there are **valid** email addresses assigned to your LDAP users, and that this tool is reading them? If it encounters a user with a non-empty but invalidly formatted email address that could cause it, so if it only happens with one user I'd inspect their object attributes to ensure they're valid.

**Comment by beharbunjaku** [comment]
This error occurred to me as well. In my case it was the "ghost" user from gitlab that was causing it.
Try adding `userNamesToIgnore: ["ghost"]`. If it is not that, check the [API logs](https://docs.gitlab.com/ee/administration/logs.html#api_jsonlog).
I think, this tool should ignore the "ghost" by default, just as it does with the "root" user.

**Comment by Adambean** [comment]
@beharbunjaku's change has been merged in if you'd like to try it out @frafra.

**Comment by frafra** [comment]
I am sorry, I am unable to test it.
